packages/lab/src/carousel/index.ts

```diff
@@ -1,2 +1,5 @@
 export * from "./Carousel";
 export * from "./CarouselSlide";
+export * from "./CarouselSlider";
+export * from "./CarouselNavigationBar";
+export * from "./CarouselControls";
```

Comment:
Does this component need to exist or be exported?
CarouselNavigationBar and CarouselControls seem like the same component?